src/features/advanced/cxpo-auto-price.ts

```
@@ -0,0 +1,162 @@
+function onTileReady(tile: PrunTile) {
+  subscribe($$(tile.anchor, C.ComExPlaceOrderForm.form), x => onFormReady(x, tile.parameter));
+}
+
+async function onFormReady(form: HTMLElement, parameter?: string) {
+  const orderBook = computed(() => cxobStore.getByTicker(parameter));
+
+  const quantityInput = await $(form.children[7], 'input');
+  const quantityValue = refValue(quantityInput);
+  const priceInfo = computed(() => {
+    const quantity = Number(quantityValue.value);
+    if (!isFinite(quantity) || quantity <= 0 || !orderBook.value) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)